kubernetes/deployment-staging.tmpl

```diff
@@ -155,6 +157,8 @@ spec:
               value: staging
             - name: SENTRY_DSN
               value: https://1f0126a750244108be76957b989081e8@sentry.io/1492498
+            - name: ENABLE_CACHE_HEADERS
+              value: true
```

Comment:
Non-blocking question just for my understanding: Does this get evaluated as a string? The conditional in the code checks for this value to be `'true'`

Reply:
my understanding is it's YAML so it does but i keep getting this wrong. See what the dry run does and then what the pod env vars are set to after deploy. Ensuring strings would be "true" i believe.

Reply:
Pretty sure numbers and booleans need to be cast to strings in templates.

Reply:
OK - please add suggestions to fix it and merge.

Reply:
```suggestion
              value: 'true'
```

Reply:
We've got bare numbers elsewhere in the template but I don't know. 🤷‍♂️

Reply:
I'm talking specifically about the setting of environment variables.